doc/admin-guide/security/index.en.rst

```diff
@@ -220,10 +220,10 @@ and origin server connections, you must do the following:
         CONFIG proxy.config.ssl.client.private_key.filename STRING "client.pem"
 
 #. Enable or disable, per your security policy, server SSL certificate
-   verification using :ts:cv:`proxy.config.ssl.client.verify.server` in
+   verification using :ts:cv:`proxy.config.ssl.client.verify.serve.policyr` in
```

Comment:
Fixing typo.  I am adding back the docs on the file-based documentations.  Since this seems to be a how-to list, I think it is ok to just discuss the new mechanism here.